### ISSUE TITLE
Bug with laser cannon dartling gunner

### ISSUE THREAD

**Issue by SwordSlicer56** [Timotheeee/btd6_mods#34]
Hello, thanks for making this mod, it is super fun! I've been playing this mod with the ultimate crosspathing mod and I noticed that my game glitches out completely whenever I get a laser cannon dartling gunner. Whenever the laser cannon is randomly put on screen, all the towers begin to bug out as well as the projectiles. Some towers stop attacking, spike factories sometimes don't shoot out spikes, but when they do the spikes glitch out and sometimes even grow in size strangely. Aces drop darts on the ground for some reason, and the game becomes basically unplayable. I'm not sure if this is because of the Ultimate Crosspathing mod but just thought I would point this bug out, because it's quite strange lol.

**Comment by SapphireSage7** [comment]
Yup, dartling gunners are completely bugged. If you upgrade from a spike storm into hydra rocket pods, the dartling gunner will constantly shoot out spike storms
